OpenRA.Utility/Program.cs

```diff
@@ -15,6 +15,16 @@
 
 namespace OpenRA.Utility
 {
+	public class NoSuchCommandException : ArgumentException
```

Comment:
`ArgumentException` defines `ParamName` which hasn't been set at all here, so either that's an oversight or `ArgumentException` doesn't apply in the first place.